sdk/identity/identity/review/identity.api.md

```diff
@@ -314,7 +327,7 @@ export interface UsernamePasswordCredentialOptions extends TokenCredentialOption
 // @public
 export class VisualStudioCodeCredential implements TokenCredential {
     constructor(options?: VisualStudioCodeCredentialOptions);
-    getToken(scopes: string | string[], _options?: GetTokenOptions): Promise<AccessToken>;
+    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken>;
```

Comment:
Is this a breaking change? Can callers specify the argument name explicitly?

Reply:
This is not a breaking change, just a name change :)

Reply:
Callers can’t specify the argument names like this, they just pass the value.